tests/sentry/dynamic_sampling/test_adjusments_models.py

```diff
@@ -0,0 +1,90 @@
+from operator import attrgetter
+
+from sentry.dynamic_sampling.models.adjustment_models import AdjustedModel
+from sentry.dynamic_sampling.models.adjustment_models import Project as P
+
+
+def test_adjust_sample_rates_org_wo_projects():
```

Comment:
nit: Can we name this `with_no_projects`? Just to make it simpler to read.

Reply:
Maybe you missed this comment, but you can keep also `wo`. It's just for consistency with the other names that I proposed `with_no`.

Reply:
@iambriccardo fixed.